packages/cli/src/commands/sandbox/sandbox-secret/sandbox_secret_remove_command.ts

```diff
@@ -41,28 +42,53 @@ export class SandboxSecretRemoveCommand
     const sandboxBackendIdentifier = await this.sandboxIdResolver.resolve(
       args.identifier
     );
-    await this.secretClient.removeSecret(
-      sandboxBackendIdentifier,
-      args.secretName
-    );
-
-    printer.print(`Successfully removed secret ${args.secretName}`);
+    if (args.secretName) {
+      await this.secretClient.removeSecret(
+        sandboxBackendIdentifier,
+        args.secretName
+      );
+      printer.print(`Successfully removed secret ${args.secretName}`);
+    } else if (args.all) {
+      const secrets = await this.secretClient.listSecrets(
+        sandboxBackendIdentifier
+      );
+      await Promise.all(
+        secrets.map((secret) =>
+          this.secretClient.removeSecret(sandboxBackendIdentifier, secret.name)
+        )
+      );
```

Comment:
The delete api has low TPS quota https://docs.aws.amazon.com/general/latest/gr/ssm.html#parameter-store .

I think we should add `secretClients.removeSecrets(backendIndentifer, secretNames: Array<string>)`

And use `DeleteParameters` batch api instead of this.